src/cmd/cli/command/compose.go

```diff
@@ -404,6 +414,8 @@ func makeComposeLogsCmd() *cobra.Command {
 	logsCmd.Flags().BoolP("raw", "r", false, "show raw (unparsed) logs")
 	logsCmd.Flags().StringP("since", "S", "", "show logs since duration/time")
 	logsCmd.Flags().Bool("utc", false, "show logs in UTC timezone (ie. TZ=UTC)")
+	logsCmd.Flags().String("type", "ALL", "show logs of type: [ALL RUN BUILD]")
```

Comment:
suggest to make this a Cobra option type. Look at some of the others on how to do it, eg. `provider`, `color`..